p4-16/spec/P4-16-spec.mdk

```diff
@@ -3057,6 +3058,7 @@ typeArg
     : DONTCARE
     | typeRef
     | nonTypeName
+    | VOID
```

Comment:
@mbudiu-vmw Mihai took the action item in LDWG meeting to think about why VOID is here, and also the occurrence in the production of the `realTypeArg` nonterminal of the grammar.

Reply:
Perhaps this may make sense for some extern objects that could be instantiated with "void" for some type parameters.